lib/graphql/list_type.rb

```diff
@@ -17,8 +17,13 @@ def to_s
     "[#{of_type.to_s}]"
   end
 
-  def coerce_input(value)
-    inner_type = of_type
-    value.map { |item| inner_type.coerce_input!(item) }
+  def valid_non_null_input?(value)
+    value = [value] unless value.is_a?(Array)
```

Comment:
http://facebook.github.io/graphql/#sec-Lists

> If the value passed as an input to a list type is not as list, it should be coerced as though the input was a list of size one, where the value passed is the only item in the list. This is to allow inputs that accept a “var args” to declare their input type as a list; if only one argument is passed (a common case), the client can just pass that value rather than constructing the list.